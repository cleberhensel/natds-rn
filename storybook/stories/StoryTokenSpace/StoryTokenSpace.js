/* eslint-disable max-lines */
import React from 'react';
import {
  ScrollView,
  SafeAreaView,
  Dimensions,
} from 'react-native';
import {
  NatContainer,
  NatSpace,
  ThemeNaturaLight as Theme,
  NatText,
} from '../../../lib';

const styles = {
  defaultScreen: {
    flex: 1,
    width: Dimensions.get('window').width,
  },
};

export const StoryTokenSpace = () => (
  <SafeAreaView style={styles.defaultScreen}>
    <NatContainer>
      <ScrollView>
        <NatSpace
          marginTop="small"
          marginBottom="none"
          marginRight="none"
          marginLeft="none"
          paddingTop="none"
          paddingBottom="none"
          paddingRight="none"
          paddingLeft="none"
          backgroundColor={Theme.palette.primary.main}
        />
        <NatText value="None" type="body1" />
        <NatSpace
          marginTop="small"
          marginBottom="none"
          marginRight="none"
          marginLeft="none"
          paddingTop="none"
          paddingBottom="micro"
          paddingRight="none"
          paddingLeft="none"
          backgroundColor={Theme.palette.primary.main}
        />
        <NatText value="Micro" type="body1" />
        <NatSpace
          marginTop="small"
          marginBottom="none"
          marginRight="none"
          marginLeft="none"
          paddingTop="none"
          paddingBottom="tiny"
          paddingRight="none"
          paddingLeft="none"
          backgroundColor={Theme.palette.primary.main}
        />
        <NatText value="Tiny" type="body1" />
        <NatSpace
          marginTop="small"
          marginBottom="none"
          marginRight="none"
          marginLeft="none"
          paddingTop="none"
          paddingBottom="small"
          paddingRight="none"
          paddingLeft="none"
          backgroundColor={Theme.palette.primary.main}
        />
        <NatText value="Small" type="body1" />
        <NatSpace
          marginTop="small"
          marginBottom="none"
          marginRight="none"
          marginLeft="none"
          paddingTop="none"
          paddingBottom="standard"
          paddingRight="none"
          paddingLeft="none"
          backgroundColor={Theme.palette.primary.main}
        />
        <NatText value="Standard" type="body1" />
        <NatSpace
          marginTop="small"
          marginBottom="none"
          marginRight="none"
          marginLeft="none"
          paddingTop="none"
          paddingBottom="semi"
          paddingRight="none"
          paddingLeft="none"
          backgroundColor={Theme.palette.primary.main}
        />
        <NatText value="Semi" type="body1" />
        <NatSpace
          marginTop="small"
          marginBottom="none"
          marginRight="none"
          marginLeft="none"
          paddingTop="none"
          paddingBottom="medium"
          paddingRight="none"
          paddingLeft="none"
          backgroundColor={Theme.palette.primary.main}
        />
        <NatText value="Medium" type="body1" />
        <NatSpace
          marginTop="small"
          marginBottom="none"
          marginRight="none"
          marginLeft="none"
          paddingTop="none"
          paddingBottom="huge"
          paddingRight="none"
          paddingLeft="none"
          backgroundColor={Theme.palette.primary.main}
        />
        <NatText value="Huge" type="body1" />
      </ScrollView>
    </NatContainer>
  </SafeAreaView>

);
