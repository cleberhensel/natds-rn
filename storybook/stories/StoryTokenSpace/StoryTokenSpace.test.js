import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components/native';

import { StoryTokenSpace } from './StoryTokenSpace';

describe('StoryTokenSpace component', () => {
  it('Should render StoryTokenSpace component ', () => {
    const button = renderer.create(
      <StoryTokenSpace />,
    ).toJSON();
    expect(button).toMatchSnapshot();
  });
});
