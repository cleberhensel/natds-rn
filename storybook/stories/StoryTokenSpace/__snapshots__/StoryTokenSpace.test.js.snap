// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoryTokenSpace component Should render StoryTokenSpace component  1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View
    style={
      Array [
        Object {
<<<<<<< HEAD
          "backgroundColor": "#FFFFFF",
=======
          "backgroundColor": "white",
>>>>>>> origin/update/react
          "bottom": 0,
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "left": 0,
          "paddingBottom": 0,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 0,
          "right": 0,
          "top": 0,
        },
        Object {
          "backgroundColor": "#FAFAFA",
          "marginTop": 0,
          "width": 750,
        },
      ]
    }
  >
    <RCTScrollView>
      <View>
        <View
          backgroundColor="#F4AB34"
          marginBottom="none"
          marginLeft="none"
          marginRight="none"
          marginTop="small"
          paddingBottom="none"
          paddingLeft="none"
          paddingRight="none"
          paddingTop="none"
          style={
            Array [
              Object {
                "backgroundColor": "#F4AB34",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 16,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        />
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            None
          </Text>
        </View>
        <View
          backgroundColor="#F4AB34"
          marginBottom="none"
          marginLeft="none"
          marginRight="none"
          marginTop="small"
          paddingBottom="micro"
          paddingLeft="none"
          paddingRight="none"
          paddingTop="none"
          style={
            Array [
              Object {
                "backgroundColor": "#F4AB34",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 16,
                "paddingBottom": 4,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        />
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            Micro
          </Text>
        </View>
        <View
          backgroundColor="#F4AB34"
          marginBottom="none"
          marginLeft="none"
          marginRight="none"
          marginTop="small"
          paddingBottom="tiny"
          paddingLeft="none"
          paddingRight="none"
          paddingTop="none"
          style={
            Array [
              Object {
                "backgroundColor": "#F4AB34",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 16,
                "paddingBottom": 8,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        />
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            Tiny
          </Text>
        </View>
        <View
          backgroundColor="#F4AB34"
          marginBottom="none"
          marginLeft="none"
          marginRight="none"
          marginTop="small"
          paddingBottom="small"
          paddingLeft="none"
          paddingRight="none"
          paddingTop="none"
          style={
            Array [
              Object {
                "backgroundColor": "#F4AB34",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 16,
                "paddingBottom": 16,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        />
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            Small
          </Text>
        </View>
        <View
          backgroundColor="#F4AB34"
          marginBottom="none"
          marginLeft="none"
          marginRight="none"
          marginTop="small"
          paddingBottom="standard"
          paddingLeft="none"
          paddingRight="none"
          paddingTop="none"
          style={
            Array [
              Object {
                "backgroundColor": "#F4AB34",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 16,
                "paddingBottom": 24,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        />
        <View
          margin={24}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
                "marginBottom": 24,
                "marginLeft": 0,
                "marginRight": 0,
                "marginTop": 0,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
              },
            ]
          }
        >
          <Text
            color="#333333"
            style={
              Array [
                Object {
                  "backgroundColor": "#FFF",
                  "color": "#333333",
                  "fontSize": 16,
                  "fontWeight": "400",
                  "lineHeight": 24,
                },
              ]
            }
            type="body1"
          >
            Standard
          </Text>
        </View>
      </View>
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
